packages/smart-contracts/src/contracts/InvoiceNFT.sol

```diff
@@ -0,0 +1,121 @@
+// Contract based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
```

Comment:
Please rename Smart Contract to leave room for other payment methods

For example: `ERC20InvoiceNFT`, `ERC20ConversionInvoiceNFT`, and `ETHInvoiceNFT`

Reply:
Will rename to `ERC20InvoiceNFT`, thanks